.github/workflows/run-tests.yml

```
@@ -0,0 +1,72 @@
+on:
+  push:
+  pull_request:
+  schedule:
+  - cron: '0 0 * * *'
+
+jobs:
+  php-tests:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php: [7.4, 7.3, 7.2]
+        laravel: [7.*]
+        testbench: [5.*]
+        dependency-version: [prefer-lowest, prefer-stable]
+        include:
+          - laravel: 7.*
+            testbench: 5.*
+            php: 7.4
+            dependency-version: prefer-stable
+          - laravel: 6.*
+            testbench: 4.*
+            php: 7.3
+            dependency-version: prefer-stable
+          - laravel: 5.8.*
+            testbench: 3.8.*
+            php: 7.3
+            dependency-version: prefer-stable
+          - laravel: 5.7.*
+            testbench: 3.7.*
+            php: 7.2
+            dependency-version: prefer-stable
+          - laravel: 5.6.*
+            testbench: 3.6.*
+            php: 7.2
+            dependency-version: prefer-stable
+          - laravel: 5.5.*
+            testbench: 3.5.*
+            php: 7.2
+            dependency-version: prefer-stable
+        exclude:
+          - laravel: 5.7.*
+            php: 7.4
+          - laravel: 5.6.*
+            php: 7.4
+          - laravel: 5.5.*
+            php: 7.4
+
+    name: P${{ matrix.php }} - L${{ matrix.laravel }} - ${{ matrix.dependency-version }} - ${{ matrix.os }}
```

There's no variation in the matrix.os?
```suggestion
    name: P${{ matrix.php }} - L${{ matrix.laravel }} - ${{ matrix.dependency-version }}
```